src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -375,6 +377,113 @@ describe('<ReviewCollapsibleChapter>', () => {
     });
   });
 
+  it('should show single page title when != chapter title', () => {
+    const testPageTitle = 'test page title';
+    const testChapterTitle = 'test chapter title';
+
+    const onEdit = sinon.spy();
+    const pages = [
+      {
+        pageKey: 'test',
+        title: testPageTitle,
+        updateFormData: (oldData, newData) => ({ ...newData, bar: 'baz' }),
+      },
+    ];
+    const chapterKey = 'test';
+    const chapter = {
+      title: 'test chapter title',
```

Comment:
Could use the `testChapterTitle` variable here too.

Reply:
fixed